src/Goteo/Controller/Dashboard/TranslateProjectDashboardController.php

```diff
@@ -87,48 +78,24 @@ public function validateProject($pid = null, $section = 'summary', $lang = null,
                 $story = reset(Stories::getall(false, false, ['project' => $this->project->id]));
                 $data['percents']['story']= $story ? $story->getLangsGroupPercent($lang_check, ['id']) : 0;
             }
-
         }
 
         View::getEngine()->useData($data);
         return $project;
     }
 
-    /**
-     * Some handy defaults for the form
-     */
-    public function createFormBuilder($defaults = null, $name = 'autoform', array $options = ['attr' => ['class' => 'autoform hide-help']]) {
+    public function createFormBuilder(
+        $defaults = null,
+        $name = 'autoform',
+        array $options = ['attr' => ['class' => 'autoform hide-help']]
+    ): FormBuilder {
         return parent::createFormBuilder($defaults, $name, $options);
     }
 
-    /**
-     * Handy method to get a form builder
-     * @return Goteo\Library\Forms\FormProcessorInterface
-     */
-    public function getModelForm($form, Model $model, array $defaults = [], array $options = [], Request $request = null) {
-        $finder = $this->getService('app.forms.finder');
-        $finder->setModel($model);
-        $validate = $mock_validation = false;
-        if($request) {
-            $validate = $request->query->has('validate');
-            $mock_validation = $validate && $request->isMethod('get');
-        }
-        // $finder->setBuilder($this->createFormBuilder($defaults, 'autoform', $mock_validation ? ['csrf_protection' => false] : []));
-        // $finder->setBuilder($this->createFormBuilder($defaults));
-        // TODO: a better way to create a csrf_protection without showing errors CSRF on mock_validation
-        $finder->setBuilder($this->createFormBuilder($defaults, 'autoform', ['csrf_protection' => false, 'attr' => ['class' => 'autoform hide-help']]));
-        $processor = $finder->getInstance($form, $options);
-        // Set full validation if required in Request
-        // Do a fake submit of the form on create to test errors (only on GET requests)
-        $processor->setFullValidation($validate, $mock_validation);
-
-        return $processor;
-    }
-
```

Comment:
I've refactored Controller::getModelForm so we don't repeat code in child controllers